docs/pages/attributesJSON.rst

```diff
@@ -46,9 +46,125 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
+
+"stage_instance"
+  - json object
+  - the required Scene Object Instance parameters to create the stage in the scene.
+
+"object_instances"
+  - array of json ojbects
+  - each element of the array is a Scene Object Instance parameters required to create a single object in the scene.
+
+The JSON tags the Scene Object Instance Attributes support are as follows:
+
+"template_name"
+  - string
+  - Dataset-unique name of the template for the stage or object attributes desired to instantiate. Since attributes templates can encode full file paths, their handles can be very long; therefore, the "template_name" field supports substring lookup, and an incomplete handle can be used for this field so long as it uniquely maps to the full handle specified in the Scene Dataset Attributes.
+
+"translation_origin"
+  - string
+  - Specifies the translation origin to use for the instantiation of the specified object. Values are case-insensitive. Supports "asset_local", where the origin is defined in the individual asset's local frame, "com", where the asset's com is the origin for transformations, or "unknown" if the value is unknown. If this value is specified, and not set to "unknown", it will override the top-level default, if any exists. Currently stages do not support instantiation transformations.
+
+"motion_type"
+  - string
+  - The desired motion type of the object being instanced. Values are case-insensitive. Values supported are "static", "kinematic", and "dynamic". This field is ignored for stages, which are always created with STATIC motion type.
+
+"translation"
+  - 3-vector
+  - The translation from the origin of the scene where the object should be placed. Currently stages do not support instantiation transformations.
+
+"rotation"
+  - Quaternion (read as [w,x,y,z], with idx 0 referencing the scalar part.)
+  - The rotation to be applied to the object upon instantiation. Currently stages do not support instantiation transformations.
+
+
 `Stage Attributes`_
 ===================
-A stage in Habitat-Sim is a static object consisting of static background scenery wherein an agent acts.  Stage Attributes templates hold relevant information describing a stage's render and collision assets and physical properties.  Any source configuration files used to build these attributes should be named using the following format:
+A stage in Habitat-Sim is a static object consisting of static background scenery wherein an agent acts. Stage Attributes templates hold relevant information describing a stage's render and collision assets and physical properties. Any source configuration files used to build these attributes should be named using the following format:
```

Comment:
Ah, there's the definition I was looking for. We should accumulate all such definitions and list them at one place.